### DISCUSSION TITLE
WASM / Emscripten

### DISCUSSION THREAD

**Discussion by ErikSom** [InteractiveComputerGraphics/SPlisHSPlasH#212] [Ideas]
Have you considered making a web export to be able to run this in the browser like for instance these physics engines:

https://github.com/kripken/ammo.js/
https://github.com/jrouwe/JoltPhysics.js
https://github.com/ashconnell/physx-js

**Comment by janbender** [comment]
No, we did not consider this yet. We just have some simple javascript examples to explain the simulation methods: 
https://interactivecomputergraphics.github.io/physics-simulation/

**Comment by ErikSom** [comment]
Hi Jan,

Thanks for your answer! I think it would be a really fun avenue to explore!
I would be willing to help integrate this with an engine like PlayCanvas or
ThreeJS.
My knowledge of compiling things to emscripten is very little, however i've
been doing web game development for over 15 years and am working on some of
the biggest gaming IP's on the web (like Subway Surfers).

Kind regards,

Erik Sombroek

Op vr 11 nov. 2022 om 15:50 schreef Jan Bender ***@***.***>:
